src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figures {
+    private String color;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public String toString() {
+        return super.toString();

```

Comment:
The `toString` method is overridden here but it only calls `super.toString()`, which is the default behavior. This override is unnecessary unless you want to provide a specific implementation for the `toString` method for the `Figures` class.